tests/matlab_test_data_collectors/python_testers/test_linear_array_transducer.py

```diff
@@ -0,0 +1,101 @@
+from tests.matlab_test_data_collectors.python_testers.utils.record_reader import TestRecordReader
```

Comment:
Not something for this PR but the path to import a class is very long here. We might want to work on shortening it.

Reply:
I have created and [issue](https://github.com/waltsims/k-wave-python/issues/219) for this